guides/user/canary/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: single
+title:  "Using Spinnaker for Automated Canary Analysis"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+
+Canary is a deployment process in which a change is partially rolled out, then
+evaluated against the current deployment (baseline) to ensure that the new
+deployment is operating at least as well as the old. This evaluation is done
+using key metrics chosen when the canary is configured.
+
+Canaries are usually run against deployments containing changes to code, but they
+can also be used for operational changes, including changes to configuration.
+
+The canary process is not a substitute for other forms of testing.
+
+## Prerequisites
+
+### Instrument your code for metrics
+
+If you're going to generate the kinds of metrics that Spinnaker canary
```

Comment:
One can also use infrastructure metrics that may already be published without the service owner having to generate anything explicitly. For example, on GCE various metrics are automatically published to stackdriver if stackdriver is enabled. You can then install a monitoring agent to collect more comprehensive metrics (including for various well known software packages). Application or service-level metrics are another step beyond this (and these are the ones you're already describing here). All of these things can be used to make judgments.